### DISCUSSION TITLE
제페토 관련 유니티 교육 문의

### DISCUSSION THREAD

**Discussion by Ellie2050** [naverz/zepeto-studio-kor#162] [4. General]
안녕하세요. 유니티 초보입니다. 유니티로 맵 준비를 하고 있는데,, 독학으로 하고 있어요.
많이 답답하고 어렵고 그렇습니다. 
학원다니면 c#으로 시작해 유니티 전체를 배워서 좋긴하겠지만,, 혹시 제페토 전용으로 타입스크립트를 비롯해서 제페토에서 필요한 부분만 배우고 싶은데 관련 교육이나 내용 등에 대해 아시는 분 계실까요? 
일반 컴퓨터학원에서는 제페토 월드맵관련한 교육과정이 아직 없더라구요.

**Comment by x4cf** [comment]
아직은 제페토 가이드와 깃의 정보가 전부입니다. 깃 또는 저희 방에서 편하게 질문주세요 :)
